services/fee-estimator/shared/dynamicFeesUtils.js

```diff
@@ -129,6 +129,7 @@ const getEstimateFeeByte = async () => { // aka getBestEstimateAvailable
 		status: 'SERVICE_UNAVAILABLE',
 	};
 };
+
 Signals.get('newBlock').add(async () => {
```

Comment:
Could you please refactor the anonymous listener function (from within the `Signals.get('newBlock').add(...)`) to be a named function and then register it as a named-listener, just like everywhere else?
We do not want any anonymous listeners registered to the signals.

Reply:
Applied